Annotate container image with github source

gberche-orange opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our contributor guidelines: https://github.com/kudobuilder/kuttl/blob/main/CONTRIBUTING.md
2. Make sure you have added and ran the tests before submitting your PR
3. If the PR is unfinished, start it as a Draft PR: https://github.blog/2019-02-14-introducing-draft-pull-requests/
-->

**What this PR does / why we need it**:


<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes https://github.com/kudobuilder/kuttl/issues/390

kensipe commented:
This seems pretty straight forward... @gberche-orange could you sign your commit for DCO?

gberche-orange commented:
Thanks @kensipe 

Sorry for the lack of DCO signoff from my commit originated from github web ui. This is now fixed.

ps: I've rebased the commit (`git rebase --signoff HEAD~1`) and configured my works to require sign-offs for
*  next github web commits ( see related https://github.blog/changelog/2022-06-08-admins-can-require-sign-off-on-web-based-commits/) 
* next intellij-based commits (see https://www.jetbrains.com/help/idea/commit-changes-dialog.html#before_commit)

So, hopefully my next commits won't be missing DCO again.

gberche-orange commented:
It might be possible that if the `/kudobuilder/kuttl` repo or `/kudobuilder` org enables the `Require contributors to sign off on web-based commits` option, then any new fork would automatically inherit from this option, and offer by default to sign off commits made through GH web UI.

kensipe commented:
thanks for patience... I've taken to long to get to this.  I will review tomorrow this and the other PRs... really appreciate the contributions!

erikgb commented on Dockerfile:
```diff
@@ -27,6 +27,8 @@ RUN make cli
 # release image with kubectl + kuttl
 FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 
+LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl/"
```
```suggestion
LABEL org.opencontainers.image.source="https://github.com/kudobuilder/kuttl"
```
I am not sure what's correct, but it seems to me that it's more common to not include a trailing slash when using this label.

gberche-orange replied:
thanks @erikgb !

erikgb reviewed (approved):
\lgtm

porridge commented:
Oh no, I clicked the button to refresh the branch and now the DCO check fails! :sob:
@gberche-orange would it be too much if I asked you to please rebase your commit?

gberche-orange commented:
@porridge sure, np, rebased and squashed in https://github.com/kudobuilder/kuttl/commit/ce0a883c448171bedf6b82a620d0cb9086b9103b
